tools/src/main/scala/scala/scalanative/interflow/Opt.scala

```diff
@@ -38,6 +38,11 @@ trait Opt { self: Interflow =>
     val fresh = Fresh(0)
     val state = new State(Local(0))
 
+    // Interflow usually infers better types on our erased type system
+    // than scalac, yet we live it a benefit of the doubt and make sure
```

Comment:
Maybe: `yet we leave it as a benefit`?

Reply:
I've not seen this type when copy-pasting. Consider it fixed. Thanks for spotting this one!